src/Catalog/Product.php

```diff
@@ -23,7 +23,7 @@ class Product extends AbstractEntity
     protected float $exchangeRateRule;
     protected string $valueType;
     protected int $maxValue;
```

Comment:
I think this should likely be a float too.  Should probably bugfix this in a follow up.